Observatory-Control-System/BLOCKS/block-jira-project-workflow.rst

```diff
@@ -0,0 +1,245 @@
+* New BLOCK Jira ticket is reviewed by Commissioning Science team during one of the regular, weekly planning meetings, and accepted/rejected for scheduling. 
+* New BLOCK Jira ticket is given a priority for scheduling and a Commissioning Scientist is Assigned the ticket, taking on the responsibility of Assignee. 
+* Proposer and Assignee work together to prepare BLOCK ticket for execution.
+* BLOCK ticket is marked as READY by Assignee, and added to the queue of tasks to be executed. 
+* BLOCK ticket is planned for execution at the summit by the night planner. 
+* BLOCK ticket is executed at the summit by Observing Specialist.
+* Data is analyzed by the Proposer and ticket is reported on supporting tickets (e.g. SITCOM). 
+* Proposer verifies observing task is completed and transitions ticket to Done to close issue.
+
+BLOCK Ticket Requirements
+=========================
+
+In addition to the standard JIRA ticket fields, proposers are also required to supply the following information: 
+* Component - either be MainTel or AuxTel
+* Original Time Estimation - selectable in the More Fields section of the ticket. 
+* Links to supporting material either tickets (e.g. SITCOM/LVV) or confluence pages
+
+Time estimation is an important part of the planning process.
+Be sure to include both the execution time and any additional time spent configuring or logging the activity in your time estimate. 
+
+Description
+-----------
+
+The description section of a BLOCK ticket must be carefully filled in to ensure success of the BLOCK. 
+BLOCK descriptions walk a very fine line between having just enough information and not too much. 
+They should not read like a novel, too much information is likely to get lost at the pace with which execute at night. 
+They must contain the minimum set of instructions needed to carry out the complete observing task. 
+
+We recommend that at a minimum the description section must contain:
+* Description, high-level overview of test
+* Pre-conditions/requirements to run test, including system status
+* Environmental Constraints on test (e.g. observing conditions/time)
+* Test Execution Instructions
+* Brief description of test steps and expectations
+
+Make use of the formatting options to provide separation between sections and make this information easy to find. 
+
+Here we provide a complete example. 
+
+**Description**
+Include a brief, 1-2 sentence overview of the test here. 
+
+**Pre-conditions**
+These should be provided as a list or short descriptive section. Examples include
+* MTRotator in DISABLED state
+* EUI Recording enabled
+* TMA Settings set to 40% max
+
+**Environmental Constraints**
+This section should include any time or condition constraints e.g. 
+* Atmospheric Seeing less that 1.5 arcseconds.
+* Photometric conditions required. 
+* Test must be executed between 12:00 and 01:00 local time. 
+
+**Test Execution Instructions**
```

Comment:
Same as above, should we change this based on the adoption of Test Cases?

Reply:
See my comment below.